### DISCUSSION TITLE
Add Telegram and Veracrypt to the Linux Mint repository

### DISCUSSION THREAD

**Discussion by AlanJojo** [linuxmint/discussions#554] [Ideas]
Recently, Ubuntu repositories have stopped providing telegram-desktop in the official repositories. So even if I use Linux Mint, it is not possible for me to update Telegram without adding 3rd party PPAs.

Could you please add Telegram and Veracypt to official Linux Mint repositories?
The current PPAs I use for them are:

Telegram - ppa:atareao/telegram
Veracrypt - ppa:unit193/encryption

**Comment by Fantu** [comment]
Someone already tried to package veracrypt on Debian (and will be also on derivates) but can't be for license issue
https://wiki.debian.org/Software%20that%20can't%20be%20packaged

telegram seems already present: https://tracker.debian.org/pkg/telegram-desktop
EDIT: sorry I saw only after that on ubuntu telegram-desktop is one of the app forced to snap

**Reply by AlanJojo** [reply]
So, will we get a deb package for telegram in Linux Mint?

**Reply by Fantu** [reply]
On Debian there is, Ubuntu instead have removed it and "migrated" to snap, so I suppose is now blacklisted and without autosync from Debian and will not have it anymore in repository.
Mint is based on Ubuntu, LMDE (Linux Mint Debian Edition) on Debian, so is present only in LMDE.
So for Mint based on Ubuntu the actual option for having it are: ppa, snap and flatpak

**Reply by AlanJojo** [reply]
Thank you. Will try the ppa:atareao/telegram
